frontend/src/views/EventMaking/EventSettingPage.vue

```diff
@@ -1,99 +1,145 @@
-            <li class="md:grid grid-cols-3 gap-4">
-              <label class="md:place-self-start  md:mt-3 mx-4" for="start_date">開始日付</label>
-              <input class="md:place-self-center mx-10 py-1 px-1 my-2 w-72 border-2 border-gray-200 box-border" id="start_date" type="date" v-model="start_date">
-              <div v-if="backend_errors.start_date" class="error">{{ backend_errors.start_date[0] }}</div>
-            </li>
-            <li class="md:grid grid-cols-3 gap-4">
-              <label class="md:place-self-start  md:mt-3 mx-4" for="end_date">終了日付</label>
-              <input class="md:place-self-center mx-10 py-1 px-1 my-2 w-72 border-2 border-gray-200 box-border" id="end_date" type="date" v-model="end_date">
-              <div v-if="backend_errors.end_date" class="error">{{ backend_errors.end_date[0] }}</div>
+            <div v-if="backend_errors.start_date" class="error">{{ backend_errors.start_date[0] }}</div>
+            <li class="md:grid md:grid-cols-12 md:gap-4 md:items-center">
+              <label class="md:col-span-4 text-left px-3 py-2" for="end_date">終了日付</label>
+              <div class="md:col-span-8">
+                <input class="w-full py-2 px-3 border-2 border-gray-200 box-border" id="end_date" type="date" v-model="end_date">
+              </div>
             </li>
-            <li class="md:grid grid-cols-3 gap-4">
-              <label class="md:place-self-start  md:mt-3 md:mx-4" for="number">募集チーム数</label>
-              <input class="md:place-self-center mx-10 py-1 px-1 my-2 w-72 border-2 border-gray-200 box-border" id="number" type="number" required placeholder="募集チーム数" v-model="number">
+            <div v-if="backend_errors.end_date" class="error">{{ backend_errors.end_date[0] }}</div>
+            <li class="md:grid md:grid-cols-12 md:gap-4 md:items-center">
+              <label class="md:col-span-4 text-left px-3 py-2" for="number">募集チーム数</label>
+              <div class="md:col-span-8">
+                <input class="w-full py-2 px-3 border-2 border-gray-200 box-border" id="number" type="number" required placeholder="募集チーム数" v-model="number">
+              </div>
             </li>
-            <li class="md:grid grid-cols-3 gap-4 sm:flex sm:items-center">
-              <label class="md:place-self-start  md:mt-3 mx-4" for="Purpose">イベント目的</label>
-              <textarea class="md:place-self-center text-start py-1 px-1 my-2 w-72 h-40 border-2 border-gray-200 box-border" name="textarea" v-model="purpose_body" id="Purpose" required placeholder="Event Purpose"></textarea>
+            <li class="md:grid md:grid-cols-12 md:gap-4 md:items-center">
+              <label class="md:col-span-4 text-left px-3 py-2" for="Purpose">イベント目的</label>
+              <div class="md:col-span-8">
+                <textarea class="w-full h-40 py-2 px-3 border-2 border-gray-200 box-border" name="textarea" v-model="purpose_body" id="Purpose" required placeholder="Event Purpose"></textarea>
```

Comment:
placeholderが場所により英語だったり日本語だったりであべこべなのでどちらかに統一したいですね

```suggestion
                <textarea class="w-full h-40 py-2 px-3 border-2 border-gray-200 box-border" name="textarea" v-model="purpose_body" id="Purpose" required placeholder="イベントの目的"></textarea>
```

Reply:
修正しました。